src/main/java/lotto/LottoRank.java

```diff
@@ -0,0 +1,41 @@
+package lotto;
+
+public enum LottoRank {
+    FIFTH(3, 5000),
+    FOURTH(4, 50000),
+    THIRD(5, 1500000),
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
언더스코어를 활용하면 가독성을 높힐 수 있습니다!